x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/roles.yml

```diff
@@ -25,19 +25,19 @@ viewer:
   cluster: []
   indices:
     - names:
-        - ".siem-signals*"
-        - ".lists-*"
-        - ".items-*"
+        - '.siem-signals*'
```

Comment:
Why the change to single quote? it probably breaks the build no?

Reply:
I just undid this change. It doesn't break anything. This file has three different patterns for index names, and my editor doesn't like them.